editor/src/templates/index.html

```diff
@@ -117,11 +153,17 @@
             border: 1px solid beige;
           "
         >
+          <img
+            src="<%- UTOPIA_DOMAIN %>/editor/pyramid_@2x.png?hash=<%- UTOPIA_SHA %>"
+            height="74px"
+            alt="Utopia Logo"
+            class="utopia-logo-pyramid dark"
+          />
           <img
             src="<%- UTOPIA_DOMAIN %>/editor/pyramid_fullsize@2x.jpg?hash=<%- UTOPIA_SHA %>"
```

Comment:
could you actually duplicate the asset (or create a `light` and `dark` folder or similar) here? I will replace one of the assets with the correct dark mode one tomorrow